WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationCommentDetailViewController.swift

```diff
@@ -31,7 +33,13 @@ class NotificationCommentDetailViewController: UIViewController, NoResultsViewHo
     // In this case, use the Post to obtain Comment information.
     private var post: ReaderPost?
 
-    private var commentDetailViewController: CommentDetailViewController?
+    private var commentDetailViewController: CommentDetailViewController? {
+        guard let content, case let .commentDetails(viewController) = content  else {
+            return nil
+        }
+        return viewController
+    }
```

Comment:
It looks like `commentDetailViewController` is accessed only with an else statement. I suppose we simply switch on those cases and not need this value? Unless it is used commonly outside of changed lines

Reply:
Resolved internally